### PR TITLE
runtime: Fix generating wrong config paths

### PR DESCRIPTION
**Pull request by linxiulei**
I don't know why you changed it at 82ab3bbb9b71c77dfbb7dd3a1b790ebe9e265a1e, but it makes runtime don't search directory /etc/clear-containers/ anymore, and run "/usr/bin/cc-runtime --cc-show-default-config-paths" shows

```
/usr/share/defaults/clear-containers/configuration.toml
/usr/share/defaults/clear-containers/configuration.toml
```

regards

### CODE REVIEW

**Comment by linxiulei** [comment]
Since it violated the commit rules, I re-committed it

**Comment by erick0z** [comment]
@linxiulei, `/usr/share/defaults/clear-containers/configuration.toml` is the default location for the config file, [see issue 161](https://github.com/clearcontainers/packaging/issues/161). Config file located in `/etc/clear-containers` should override the default config.

**Comment by linxiulei** [comment]
@erick0z I noticed that, but it wasn't what I was concerned. The issue is the var "SYSCONFDIR" effects the auto-generated var "defaultSysConfRuntimeConfiguration" too, which makes `--cc-show-default-config-paths` showing `/usr/share/defaults/clear-containers/configuration.toml` twice.

And runtime by this way wouldn't search /etc/clear-containers/ anymore, something must be wrong here.

PS: A issue I don't figure it out as well that why "make install" is with  "SYSCONFDIR=/etc", but "make" is with "SYSCONFDIR=/usr/share/defaults"

**Comment by erick0z** [comment]
@linxiulei I see, let me dig into it! Thanks for spotting that! You should raise an issue for tracking this!

**Comment by jodh-intel** [comment]
Confirmed - this is a bug and does indeed stop `/etc/clear-containers/containers.toml` from being read.

**Comment by linxiulei** [comment]
@erick0z sure, I'll get used to it soon

**Comment by jodh-intel** [comment]
Hi @erick0z - @linxiulei has the minimal fix on this PR.

We could at some point consider simplifying the build further by running with `build-cc-system` which mostly does what we need:

```
$ make build-cc-system LIBEXECDIR=%{LIBEXECDIR} PREFIX=/usr VERSION=@VERSION@
```

The easiest way to test changed variables btw @linxiulei is to specify the `help` target as that shows the summary based on the variables you've specified. For example:

```
$ make help PREFIX=/foo | sed -ne '/• Summary:/,/• Variables affecting the build/ p' | head -n -1
```

@linxiulei - I've created an issue for you (#293). All you need to do now is:

- Update the `runtime/debian.rules-template` file with the same fix.
- Add the "Fixes #XXX" comment in the commit itself.
  See https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format for details.

**Comment by linxiulei** [comment]
Seems the Travis CI has problem if I push a commit that overwrites a history commit, I'd try to reopen this PR to fix it

**Comment by jodh-intel** [comment]
Hi @linxiulei - ah - you have hit Travis bug https://github.com/travis-ci/travis-ci/issues/7830.

To resolve this:

- `git rebase -i master`
- change "`pick`" to "`reword`" (to force a new commit hash).
- ` git push -f ...`

**Comment by linxiulei** [comment]
Hi @jodh-intel it works only if I reopen this PR. :- )

**Comment by jodh-intel** [comment]
ok - thanks @linxiulei.

Let's get an ack from @erick0z too before merging.

lgtm

**Comment by erick0z** [comment]
LGTM

**Review by erick0z — APPROVED** [review]

**Comment by jcvenegas** [comment]
@erick0z  please merge this if you are ok

**Pull request merged**
